lib/Twig/Template.php

```diff
@@ -573,6 +576,10 @@ protected function getAttribute($object, $item, array $arguments = array(), $typ
             $method = (string) $item;
             $call = true;
         } else {
+            if (isset($propertySandboxException)) {
+                throw $propertySandboxException;
```

Comment:
shouldn't this go after the ``$isDefinedTest`` condition ?

Reply:
Thanks! Fixed.